media-tv/plex-media-server/plex-media-server-1.23.1.4571.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit systemd unpacker
+
+DESCRIPTION="Free media library that is intended for use with a plex client"
+HOMEPAGE="https://www.plex.tv/"
+
+MY_PV="${PV}-6119e8eed"
+
+MY_URI="https://downloads.plex.tv/plex-media-server-new"
+SRC_URI="
+	amd64? ( ${MY_URI}/${MY_PV}/debian/plexmediaserver_${MY_PV}_amd64.deb )
+	x86? ( ${MY_URI}/${MY_PV}/debian/plexmediaserver_${MY_PV}_i386.deb )"
+
+LICENSE="Plex"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+RESTRICT="mirror bindist"
+IUSE="system-openssl"
+
+DEPEND="
+	acct-group/plex
+	acct-user/plex"
+RDEPEND="
+	${DEPEND}
+	system-openssl? ( dev-libs/openssl )"
+
+QA_PREBUILT="*"
+QA_MULTILIB_PATHS=(
+	"usr/lib/plexmediaserver/lib/.*"
+	"usr/lib/plexmediaserver/Resources/Python/lib/python2.7/.*"
+	"usr/lib/plexmediaserver/Resources/Python/lib/python2.7/lib-dynload/_hashlib.so"
+)
+
+S="${WORKDIR}"
+
+src_install() {
+	# Remove Debian specific files
+	rm -r "usr/share/doc" || die
+
+	# Remove shipped openssl library
+	if use system-openssl; then
+		rm usr/lib/plexmediaserver/libssl.so.1.0.0 || die
+	fi
+
+	# Add startup wrapper
+	dosbin "${FILESDIR}/start_pms"
+
+	# Copy main files over to image and preserve permissions so it is portable
+	cp -rp usr/ "${ED}" || die
+
+	# Make sure the logging directory is created
+	keepdir /var/log/pms
+	fowners plex:plex /var/log/pms
+
+	keepdir /var/lib/plexmediaserver
+	fowners plex:plex /var/lib/plexmediaserver
+
+	doinitd "${FILESDIR}/${PN}.init.d"
+	doconfd "${FILESDIR}/${PN}.conf.d"
```

Comment:
Good to see directory gone, but this would install them as `${PN}.conf.d` rather than `${PN}`.
I previously suggested newconfd (and newinitd) to handle that.
https://devmanual.gentoo.org/function-reference/install-functions/